src/MainWindow.h

```diff
@@ -158,9 +158,10 @@ private slots:
 
 private slots:
 	void selectFindType(int);
-	void find();
-	void findString(QString);
-	void findAndReplace();
+    void hideFind();
```

Comment:
there are other inconsistencies like this already in the file. should i change these too?